packages/core/src/network/heartbeat.spec.ts

```diff
@@ -87,9 +87,9 @@ describe('unit test heartbeat', async () => {
     // Chris dies, alice heartbeats again
     alice.interaction.interact = sinon.fake((id: PeerId) => {
       if (id.equals(chris.id)) {
-        throw new Error('FAIL')
+        return Promise.resolve(-1)
```

Comment:
Shouldn't this be `Promise.reject` ?

Reply:
It simulates Ping behavior and does what heartbeat should do, return sth >= 0 on success and -1 otherwise